src/assets/i18n/fr.json

```diff
@@ -46,6 +46,7 @@
   "anms.settings.general.language.fr": "Français",
   "anms.settings.general.language.pt-br": "Portugais",
   "anms.settings.general.language.sk": "Slovaque",
+  "anms.settings.general.language.zh-cn": "",
```

Comment:
Because I did not know how 'Chinese' translated in French or other language, although I can google translate it. It only affects displaying chinese option when using other language. I can google translate it if you are ok about it

Reply:
Please check latest commit out, I've fixed that